### ISSUE TITLE
hxScout: Data received increasing, but nothing showing up in any of the views

### ISSUE THREAD

**Issue by tbrosman** [jcward/hxtelemetry#44]
I'm not sure this is a bug, probably more a request for troubleshooting ideas. I'm receiving data and I see nothing in the UI. I have gotten hxScout to work previously and am wondering if this is just the version of hxcpp/openfl/etc I have. Here is what I am seeing:

![image](https://cloud.githubusercontent.com/assets/3754050/16679960/7dc819ce-44a0-11e6-91ab-33ed71d8b694.png)

And here are the (relevant) haxelibs I am compiling against:

hxcpp - 3.2.205
lime - 2.8.2
openfl - 3.6.1
hxtelemetry - 0.3.3

I've tried changing around the hxcpp version: I started with 3.2.193 and upgrading didn't seem to make any difference.

My application.xml contains the following:

```
    <haxedef name="HXCPP_STACK_TRACE" />
    <haxedef name="HXCPP_TELEMETRY" />

    <haxelib name="hxtelemetry" />
    <config:hxtelemetry allocations="true" host="localhost" />

    <haxeflag name="-D advanced-telemetry" if="flash" />
    <haxeflag name="-D telemetry" />
```

I think the first two are redundant with -D telemetry (not sure). Any ideas what might be wrong?

**Comment by jcward** [comment]
Interesting. The [advance_frame](https://github.com/jcward/hxtelemetry/blob/master/hxtelemetry/HxTelemetry.hx#L161) function in hxtelemetry causes the graph to move forward. It typically is called from [Stage](https://github.com/openfl/openfl/blob/develop/openfl/display/Stage.hx#L742) via the [openfl.telemetry.Telemetry](https://github.com/openfl/openfl/blob/develop/openfl/profiler/Telemetry.hx#L53) class.

First you might verify that function is being reliably called in your application. The other thing that sometimes happens is too much telemetry data (e.g. too many allocations being recorded) cause delays / crashes of the UI. Try disabling allocation tracking.

**Comment by tbrosman** [comment]
I compiled with allocation tracking turned off and now it works. Thanks!

Now, real question is is there a way to get the UI to handle allocation telemetry within some reasonable level-of-detail? Or, is there a way to throttle/restrict which allocations get tracked in my app somehow? How frequently does allocation information get sent, just every frame?

**Comment by jcward** [comment]
Indeed, I've been pondering how to filter allocations, but nothing exists yet. If I recall, @larsiusprime had some issue where tons of text / XML processing made it bog during setup-time. I think you can selectively disable allocation tracking, maybe turn it on once the game has entered some menu state. Maybe try this:

```
    untyped __global__.__hxcpp_hxt_ignore_allocs(1);
```

And later enable:

```
    untyped __global__.__hxcpp_hxt_ignore_allocs(-1);
```
